packages/members-api/lib/services/stripe-webhook.js

```diff
@@ -141,7 +141,12 @@ module.exports = class StripeWebhookService {
         const member = await this._memberRepository.get({
             customer_id: subscription.customer
         });
-
+        const subscriptionPriceData = _.get(subscription, 'items.data');
+        if (subscriptionPriceData && subscriptionPriceData.length > 1) {
```

Comment:
Should this be `if (!subscriptionPriceData || subscriptionPriceData.length > 1)` :thinking: ?

Reply:
🤔 Hmm yea good point, I was only thinking for the case with more than one price data, but having 0 price is a bad case as well so makes sense to handle it and upload error message

Reply:
Yeah - in fact I think `if (!subscriptionPriceData || subscriptionPriceData.length !== 1)` is probably the best!

Reply:
Agreed, will update, thanks for quick review 🙌